### ISSUE TITLE
[Meta] TC39 Proposal Usability Studies

### ISSUE THREAD

**Issue by codehag** [tc39/tg5#29]
This is a meta issue for all [tc39 proposals](https://github.com/tc39/proposals). Which ones should we consider doing a usability study on? Open a new issue with that specific study, and link it here. 

Current:
* none

Paused:
* [Do expressions](https://github.com/tc39/tg5/issues/28)

**Comment by js-choi** [comment]
Is the old pipe operator study published online? Could it be linked here?

**Comment by codehag** [comment]
No, it isn't. I can send you the data though if you like?

**Comment by js-choi** [comment]
@codehag: Yes, that would be great. Someone at the February TG5 meeting suggested studying the pipe operator again, which could be a good idea. But we definitely should properly review the results of the old pipe study first.

I think that it would be most desirable to publish the info publicly somewhere, such as in this repository.

But if that’s not possible, please send the info to me at <jschoi@jschoi.org> or on Matrix at @jschoi:matrix.org.
